omnibus/config/software/updater.rb

```diff
@@ -52,22 +53,26 @@
 
     copy 'bin/updater', "#{install_dir}/bin/"
 
-    # Systemd
+    # Add updater unit
     systemdPath = "/lib/systemd/system/"
     if not debian_target?
       mkdir "/usr/lib/systemd/system/"
       systemdPath = "/usr/lib/systemd/system/"
     end
+    erb source: "datadog-updater.service.erb",
+       dest: systemdPath + "datadog-updater.service",
+       mode: 0644,
+       vars: { install_dir: install_dir, etc_dir: etc_dir}
 
-    # Add stable systemd units
+
+    systemdPath = "#{install_dir}/systemd/"
+    # Add stable agent units
     templateToFile = {
       "datadog-agent.service.erb" => "datadog-agent.service",
       "datadog-agent-trace.service.erb" => "datadog-agent-trace.service",
       "datadog-agent-process.service.erb" => "datadog-agent-process.service",
       "datadog-agent-security.service.erb" => "datadog-agent-security.service",
       "datadog-agent-sysprobe.service.erb" => "datadog-agent-sysprobe.service",
```

Comment:
If I'm reading the diff correctly, those files aren't packaged anymore. Do they still need to be generated?

Reply:
Yes, they are actually packaged in /opt/datadog/updater/systemd (install_dir/systemd)
Since the whole install_dir is added, files in it don't have to be declared in omnibus/config/project/udpater.rb